diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter2/KdocCommentsWarnTest.kt

```diff
@@ -9,16 +12,49 @@ import org.cqfn.diktat.ruleset.constants.Warnings.MISSING_KDOC_TOP_LEVEL
 import org.cqfn.diktat.ruleset.rules.DIKTAT_RULE_SET_ID
 import org.cqfn.diktat.ruleset.rules.chapter2.kdoc.KdocComments
 import org.cqfn.diktat.util.LintTestBase
-
-import com.pinterest.ktlint.core.LintError
-import generated.WarningNames
 import org.junit.jupiter.api.Tag
 import org.junit.jupiter.api.Tags
 import org.junit.jupiter.api.Test
 
 class KdocCommentsWarnTest : LintTestBase(::KdocComments) {
     private val ruleId: String = "$DIKTAT_RULE_SET_ID:${KdocComments.NAME_ID}"
 
+    @Test
+    @Tag(WarningNames.COMMENTED_BY_KDOC)
+    fun `Should warn if kdoc comment is inside code block`() {
+        lintMethod(
+            """
+                    |package org.cqfn.diktat.example
+                    |
+                    |/**
+                    |  * right place for kdoc
+                    |  */
+                    |class Example {
```

Comment:
could you please also check top-level methods (in Kotlin you can do it without a class and from AST perspective they can differ)?

```
fun foo() {
    /**
    */
}
```

Reply:
done